run.sh

```diff
@@ -24,5 +24,4 @@ create)
     ;;
 esac
 bundle exec rake import:all
```

Comment:
imagine if you start the container and it imports everything (from the file that's checked into the repo and therefore potentially v. out of date)
then the user deletes some courts, changes others, etc.
the we restart the container
and the deleted courts are re-imported

This should be moved into the rake db:seed task, which is inherently a one-time setup task
We should do that kind of task manually, when a new environment is set up, not every time it's run